portality/view/doaj.py

```diff
@@ -58,15 +58,15 @@ def news():
     news = blog.News.latest(app.config.get("NEWS_PAGE_NEWS_ITEMS", 20))
     return render_template('doaj/news.html', news=news, blog_url=app.config.get("BLOG_URL"))
 
+@blueprint.route("/widgets_demo")
+def widgets_demo():
+    return render_template('doaj/fqw_demo.html',
```

Comment:
This page does include some of the main search template on the right, which should be hidden as there aren't any facets.

![Screenshot from 2021-03-15 14-39-18](https://user-images.githubusercontent.com/856138/111170897-54db9080-859c-11eb-8a6c-7a90e5eaab37.png)

Reply:
Styles for the pager are also a bit wonky, but that might not be worth looking at until Sophy has done a pass over it.

Reply:
I left it for now and I will just update that to the final version. I will create the separate template for the widget to make sure this is really separated from the public search template